heat_and_trees/heat_and_trees.ipynb

```diff
@@ -446,7 +427,7 @@
    "source": [
     "## Calculate land surface temperature\n",
     "\n",
-    "Given the NDVI calculated above, we can determine land surface temperature. For ease, we'll use some *top of atmosphere* calculations that have already been written up as Python functions as part of rasterio work in the `rio_toa` module. We'll also need to specify one more for transforming satellite temperature (brightness temp) to land surface temperature. For these calculations we'll use both Thermal Infrared bands - 10 and 11."
+    "Given the NDVI calculated above, we can determine land surface temperature. For ease, we'll use some *top of atmosphere* calculations that have already been written up as Python functions as part of rasterio work in the [rio_toa](https://github.com/mapbox/rio-toa/tree/master) module. We will reproduce the `brightness_temp` and `toa_utils` functions from the `rio_toa` module here to make it easier and faster."
```

Comment:
You mention the `toa_utils` function? Where is it?

Reply:
That's a mistake. It's supposed to be the `temp_rescale` function. I will correct it now